schema.graphql

```diff
@@ -54,19 +55,21 @@ type Rental @entity(immutable: false) {
   start_date: BigInt! #TODO: change to camel case
   startedTxHash: String!
   expiration_date: BigInt #TODO: change to camel case
-  expiredTxHash: String
+  ended_at: BigInt
```

Comment:
Is this field used anywhere? If not, let's use camel case (ops, I think I suggested `ended_at` before)
```suggestion
  endedAt: BigInt
```